client/blocks/reader-featured-video/index.jsx

```diff
@@ -55,6 +62,12 @@ class ReaderFeaturedVideo extends React.Component {
 
 	handleThumbnailClick = ( e ) => {
 		e.preventDefault();
+		this.props.onThumbnailClick();
```

Comment:
In the combined card, we'll have `allowPlaying` set to false but we'll still want to execute a click handler when the thumb is clicked (to open full post view), so I thought it was best that we run `this.props.onThumbnailClick();` in all cases.